### PR TITLE
Added inspect_points operation

### PR DESCRIPTION
**Pull request by jlstevens**
This PR requires some of the fixes in https://github.com/holoviz/holoviews/pull/4792 and is essentially a way for me to continue work on https://github.com/holoviz/holoviews/pull/4794 until the aforementioned PR is merged.

### CODE REVIEW

File: holoviews/operation/datashader.py [hunk 1]
```diff
@@ -1725,3 +1725,125 @@ class directly_connect_edges(_connect_edges, connect_edges):
...
+
+    # Stream values and overrides
+    streams = param.List(default=[PointerXY])
+    x = param.Number(default=0)
+    y = param.Number(default=0)
+
+    def _process(self, raster, key=None):
+        no_match = Points(self.p.hits_transformer(None), vdims=self.p.vdims)
+        try:
+            if isinstance(raster, RGB):
```
Comment (philippjfr) [inline holoviews/operation/datashader.py:1789]: Shouldn't happen for datashaded images but not all RGBs have an alpha channel.

Comment (jlstevens) [reply holoviews/operation/datashader.py:1789]: Good point. I'll make sure to check the channel number. If alpha is missing, that would be an error imho as I don't know what else you could do (maybe say white or black are null values but then you wouldn't have a single scalar, unless you use a hex color or something).

Comment (jbednar) [reply holoviews/operation/datashader.py:1789]: I guess that could be a warning, and then just treat it as if the count is non-zero and non-NaN.

**Comment by jlstevens** [comment]
Closing now that #4794 is rebased off master with the `dask_spatialpandas` PR merged.

File: holoviews/operation/datashader.py (hunk 1 above)
Comment (jlstevens) [reply holoviews/operation/datashader.py:1789]: In other words, disable the sparseness optimization and always run the spatial queries?

Comment (jbednar) [reply holoviews/operation/datashader.py:1789]: Right.

**Pull request closed without merging**